Ruby/Ruby.sublime-syntax

```diff
@@ -742,8 +815,8 @@ contexts:
     - include: try-regex
 
   method:
-    - match: \b(def)\b
-      scope: meta.function.ruby keyword.control.def.ruby
+    - match: \bdef\b
+      scope: meta.function.ruby support.type.function.ruby
```

Comment:
`keyword.declaration.function`

Reply:
@wbond suggested `storage.type.function` for `sub` in Perl and `def` is scoped exactly like that in Python. Many other syntaxes use this scope for general purpose function declaration keywords and it matches the scope names for `namespace`, `module`, `class`, ... . Therefore it is done here as well.

Reply:
See my other comments.